akka-docs/src/main/paradox/typed/distributed-data.md

```diff
@@ -40,7 +40,7 @@ out-of-date value.
 
 ## Using the Replicator
 
-The @apidoc[typed.*.Replicator]
+The @apidoc[typed.*.Replicator$]
```

Comment:
This fixes the 404, but the linked page is not `providing the API`, only the messages. Can we link to the non-typed page?

Reply:
You're right. I think the wording was confusing for typed, as the messages are what's interesting here. I've tried rewording this bit, what do you think?

Reply:
Yes, makes more sense, thanks :)